data/fields/mtb/scale/imba.json

```diff
@@ -5,11 +5,11 @@
     "placeholder": "Easy, Medium, Difficult...",
     "strings": {
         "options": {
-            "0": "Easiest (white circle)",
-            "1": "Easy (green circle)",
-            "2": "Medium (blue square)",
-            "3": "Difficult (black diamond)",
-            "4": "Extremely Difficult (double black diamond)"
+            "0": "⚪ Easiest (white circle)",
+            "1": "🟢 Easy (green circle)",
+            "2": "🟦 Medium (blue square)",
```

Comment:
FYI, some older operating systems may not be able to display these emoji, which were introduced in Unicode 12.0 (released in 2019).

Reply:
I think that's okay because the information is still present in the text, happy to remove the emoji again if ideditor/schema-builder#56 happens.